docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.md

```diff
@@ -7,67 +7,68 @@ import CLIRemoval from '/docs/components/warnings/_cli_removal.md'
-
-:::warning
-
-If you are also storing [Validations in GCS](./how_to_configure_a_validation_result_store_in_gcs.md) or [DataDocs in GCS](../configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.md), please ensure that the ``prefix`` values are disjoint and one is not a substring of the other.
-
-:::
+In the following example, `expectations_store_name` is set to ``expectations_GCS_store``, but it can be personalized.  You also need to change the ``store_backend`` settings. The ``class_name`` is ``TupleGCSStoreBackend``, ``project`` is your GCP project, ``bucket`` is the address of your GCS bucket, and ``prefix`` is the folder on GCS where Expectations are stored.
 
 ```yaml name="tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py configured_expectations_store_yaml"
 ```
 
+:::warning
+If you are also storing [Validations in GCS](./how_to_configure_a_validation_result_store_in_gcs.md) or [DataDocs in GCS](../configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.md), make sure that the ``prefix`` values are disjoint and one is not a substring of the other.
+:::
 
-### 4. Copy existing Expectation JSON files to the GCS bucket (This step is optional)
+## 4. Copy existing Expectation JSON files to the GCS bucket (Optional)
 
-One way to copy Expectations into GCS is by using the ``gsutil cp`` command, which is part of the Google Cloud SDK. The following example will copy one Expectation, ``my_expectation_suite`` from a local folder to the GCS bucket. Information on other ways to copy Expectation JSON files, like the Cloud Storage browser in the Google Cloud Console, can be found in the [Documentation for Google Cloud](https://cloud.google.com/storage/docs/uploading-objects).
+Use the ``gsutil cp`` command to copy Expectations into GCS. For example, the following command copies the Expectation ```my_expectation_suite`` from a local folder into a GCS bucket:
 
 ```bash name="tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py copy_expectation_command"
 ```
 
+The following confirmation message is returned:
+
 ```bash name="tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py copy_expectation_output"
 ```
 
+Additional methods for copying Expectations into GCS are available. See [Upload objects from a filesystem](https://cloud.google.com/storage/docs/uploading-objects).
 
-### 5. Confirm that the new Expectation Suites have been added
+## 5. Confirm that the new Expectation Suites have been added
 
-If you copied your existing Expectation Suites to GCS, run the following Python code to confirm that Great Expectations can find them:
+If you copied your existing Expectation Suites to GCS, run the following Python command to confirm that Great Expectations can find them:
 
 ```python
```

Comment:
Is this code identical to one of the blocks I tagged earlier?  If so, they can both be updated to use the same named snippet - but this is another one that we should flag as needing to come from an example script under test.

Reply:
@Rachel-Reverie yes, the code blocks are the same. I have added a note.